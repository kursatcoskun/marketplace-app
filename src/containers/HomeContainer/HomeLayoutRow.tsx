import React from "react";
import styled from "styled-components";
import { Row } from "antd";
import { RowProps } from "antd/lib/row";

export const HomeLayoutRow: React.FunctionComponent<RowProps> = styled(Row)`
  padding-top: 41px;
  justify-content: center;
  padding-right: 7%;
  padding-left: 3%;
  display: flex;
  flex-direction: row;
  @media only screen and (max-width: 800px) {
    flex-direction: column;
  }
`;
