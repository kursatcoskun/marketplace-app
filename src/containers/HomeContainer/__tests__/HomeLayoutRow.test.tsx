// MyComponent.test.js
import { shallow } from "enzyme";
import { HomeLayoutRow } from "../HomeLayoutRow";
describe("HomeLayoutRowComponent", () => {
  it("should render HomeLayoutRow component", () => {
    const wrapper = shallow(<HomeLayoutRow />);
  });

  it("should render initial layout", () => {
    const component = shallow(<HomeLayoutRow />);
    expect(component.getElements()).toMatchSnapshot();
  });
});
