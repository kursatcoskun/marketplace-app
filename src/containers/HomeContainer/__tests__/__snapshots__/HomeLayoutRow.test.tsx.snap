// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeLayoutRowComponent should render initial layout 1`] = `
Array [
  <Row
    className="sc-bdnxRM kIqlSG"
  />,
]
`;
